### PR TITLE
ots 0.1.0 (new formula)

### PR DESCRIPTION
**Pull request by slavovojacek**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

### CODE REVIEW

**Comment by slavovojacek** [comment]
Hmm, the "repo not old enough" error did not show up for me when running `brew audit --new ots` locally.

It's been up for 26 days now, so in theory 4 days to go. Are we OK to keep the PR open till then and merge once it's green provided there are no other outstanding issues? Thanks!

File: Formula/ots.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      tag:      "v0.0.11",
+      revision: "66e3c30934bfc867b6451b08f39693fe7d5c05c0"
+
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-ldflags",
+      "-s -w -X github.com/sniptt-official/ots/build.Version=#{version}", *std_go_args
```
Comment (SMillerDev) [inline Formula/ots.rb:15]:
```suggestion
    ldflags =  "-s -w -X github.com/sniptt-official/ots/build.Version=#{version}"
    system "go", "build", *std_go_args(ldflags: ldflags)
```

File: Formula/ots.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+class Ots < Formula
+  desc "🔐 Share end-to-end encrypted secrets with others via a one-time URL"
+  homepage "https://www.sniptt.com"
+
+  url "https://github.com/sniptt-official/ots.git",
```
Comment (SMillerDev) [inline Formula/ots.rb:5]:
```suggestion
  homepage "https://www.sniptt.com"
  url "https://github.com/sniptt-official/ots.git",
```

File: Formula/ots.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+class Ots < Formula
+  desc "🔐 Share end-to-end encrypted secrets with others via a one-time URL"
+  homepage "https://www.sniptt.com"
+
```
Comment (carlocab) [inline Formula/ots.rb:4]:
```suggestion
```

File: Formula/ots.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  url "https://github.com/sniptt-official/ots.git",
+      tag:      "v0.0.11",
+      revision: "66e3c30934bfc867b6451b08f39693fe7d5c05c0"
```
Comment (carlocab) [inline Formula/ots.rb:7]: Let's use a tarball instead of a Git clone, please.

File: Formula/ots.rb [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
```
Comment (carlocab) [inline Formula/ots.rb:8]:
```suggestion
```

File: Formula/ots.rb (hunk 1 above)
Comment (carlocab) [inline Formula/ots.rb:15]:
```suggestion
    ldflags = "-s -w -X github.com/sniptt-official/ots/build.Version=#{version}"
    system "go", "build", *std_go_args(ldflags: ldflags)
```

**Review by carlocab — COMMENTED** [review]
```
==> brew audit ots --online --new-formula
==> FAILED
ots:
  * GitHub repository too new (<30 days old)
Error: 1 problem in 1 formula detected
```
We'll want to wait until your repo is old enough. Also, please follow the steps in the PR template more carefully in the future: you'll have seen this error with `brew audit --new`.

File: Formula/ots.rb (hunk 1 above)
Comment (slavovojacek) [reply Formula/ots.rb]: Thanks, addressed!

File: Formula/ots.rb (hunk 2 above)
Comment (slavovojacek) [reply Formula/ots.rb]: Done!

File: Formula/ots.rb [hunk 6]
```diff
@@ -1,18 +1,14 @@
 class Ots < Formula
   desc "🔐 Share end-to-end encrypted secrets with others via a one-time URL"
   homepage "https://www.sniptt.com"
-
-  url "https://github.com/sniptt-official/ots.git",
-      tag:      "v0.0.11",
-      revision: "66e3c30934bfc867b6451b08f39693fe7d5c05c0"
-
+  url "https://github.com/sniptt-official/ots/archive/refs/tags/v0.0.11.tar.gz"
```
Comment (carlocab) [inline Formula/ots.rb:4]: You need a `sha256` line after this one.

**Review by SMillerDev — APPROVED** [review]

**Comment by SMillerDev** [comment]
Thanks @slavovojacek  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by slavovojacek** [comment]
Thanks guys, really awesome to make it to core!

**Pull request closed without merging**